### PR TITLE
Feature: send mail, forgot password

### PR DESCRIPTION
**Pull request by hoanghong0902**

### CODE REVIEW

**Review by DUYNGOC97 — APPROVED** [review]

**Comment by DUYNGOC97** [comment]
ok

File: src/main/java/com/codegym/aurora/controller/back_office/MailController.java
Comment (ChungNguyenVr912) [inline src/main/java/com/codegym/aurora/controller/back_office/MailController.java:1]: consider to combine with auth controller

File: src/main/java/com/codegym/aurora/controller/back_office/UserController.java [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequiredArgsConstructor
+@CrossOrigin("*")
```
Comment (ChungNguyenVr912) [inline src/main/java/com/codegym/aurora/controller/back_office/UserController.java:16]: please remove cors config here

File: src/main/java/com/codegym/aurora/repository/UserDetailRepository.java [hunk 2]
```diff
@@ -5,4 +5,5 @@
 
 
 public interface UserDetailRepository extends JpaRepository<UserDetail, Long> {
+    UserDetail findByEmail(String email);
```
Comment (ChungNguyenVr912) [inline src/main/java/com/codegym/aurora/repository/UserDetailRepository.java:8]: problem!

**Review by ChungNguyenVr912 — CHANGES_REQUESTED** [review]

**Comment by hoanghong0902** [comment]
fixed mail controller

**Pull request closed without merging**